app_server/app/models/User.js

```diff
@@ -120,6 +120,7 @@ module.exports = function(sequelize, DataTypes) {
     classMethods: {
       associate: function(models) {
         User.hasMany(models.Stream, {
+          as: 'stream',
```

Comment:
This should be plural according to the [docs](http://docs.sequelizejs.com/en/latest/api/associations/?highlight=hasMany#hasmanytarget-options).
>The alias of this model. If you provide a string, it should be plural, and will be singularized using node.inflection. 
>Defaults to the pluralized name of target
Not entirely sure how it works since I hadn't tried it out myself yet, but hope this helps.